service/resource/vaultpki/delete.go

```diff
@@ -9,7 +9,33 @@ import (
 	"github.com/giantswarm/cert-operator/service/key"
 )
 
```

Comment:
* renamed `Process*State` -> `Apply*Change` and moved up
* renamed `Get*State` -> `new*Change` and moved down